website/customers.html

```diff
@@ -1,6 +1,6 @@
 ---
-title: Redash Customers
-description: Great collaction of our users
+title: Redash customers
```

Comment:
`Redash *C*ustomers`

Although describing this as "Redash Users" is more accurate. Just not sure if it "sounds" right.